### ISSUE TITLE
Create Dual-Major Workflow

### ISSUE THREAD

**Issue by shanitajohn** [curationexperts/laevigata#148]
Create a workflow / wireframe option for Dual Major students based on the information below.

**DETAILS**
- Superuser and Developer intervention is needed if a student is going to submit 2 theses or a thesis + dissertation (i.e. they will have two separate records in the same semester) because the current application wasn't built for this use case. Ideally, the new app would handle this better.

- Superuser and Developer intervention is NOT needed if the dual degree student is only submitted 1 thesis/dissertation. The student can change their school/program/department if needed in the application to reflect which degree they are submitting a thesis/dissertation for.

**Comment by mark-dce** [comment]
We haven't been blocked by this use case in the 2 years the system has been running.  If the issue arrises, we'll attempt to handle it via manual intervention rather than trying to develop a solution that can accommodate all possible edge-cases.  If this requirement surfaces multiple times in the future, we'll open a new ticket based on the specific concrete cases we see.
